### ISSUE TITLE
sign up != log in

### ISSUE THREAD

**Issue by ackpacket** [snap-cloud/snap-cloud#68]
After you sign you you aren't logged in.

**Comment by cycomachead** [comment]
I think this is intentional behavior...at least we aren't the only site that works this way, though I'm not quite sure why this is standard. I thought it was more secure but I don't know why....

**Comment by ackpacket** [comment]
Yeah.  Kevin seemed to be bothered by it, thought I might bring it up as an issue.  If anyone can think of a good idea as to why this is intentional behavior, feel free to close the issue.

**Comment by cycomachead** [comment]
Heh, that's funny. I thought I had read somewhere that it was better for password security, but I think it could also be a spam detection thing.

**Comment by Fortisque** [comment]
If it's intentional its fine, just wanted to bring it up. Usually would do a flash or something to tell user they successfully signed up since it a bit disorienting to return to home page after sign up with no visual cues that anything happened

**Comment by cycomachead** [comment]
@Fortisque Heh, the flash thing is a bug / to be implemented....
https://www.pivotaltracker.com/n/projects/1279044/stories/92995152

Our app is kinda lacking from a UX level :stuck_out_tongue:

**Comment by cycomachead** [comment]
P.S. If you ta 169 again you should let people use github issues instead of tracker, but GH Issues are so much simpler and we never checked our velocity in tracker.

**Comment by Fortisque** [comment]
Haha, I'm done GSI'ing actually. I like GH issues as well, but I guess for standardization purposes both Necula's and Fox's version require PT. I think it's because PT is closer to the software that is used in industry. (Actually I just know that Dreamworks and GoDaddy use JIRA, and PT is pretty close to JIRA)

**Comment by cycomachead** [comment]
Hahah, I like JIRA more too, and pretty much everyone uses some form of an issue tracker.

I like that GH issues works super easy with the github command line tool too.

**Comment by cycomachead** [comment]
Also reference 
#51 #26
